gymnasium_robotics/envs/multiagent_mujoco/obsk.py

```diff
@@ -0,0 +1,1110 @@
+from __future__ import annotations
+
+import itertools
+import typing
+from copy import deepcopy
+
+import numpy as np
+
+
+class Node:
+    """A node of the mujoco graph representing a single body part and it's corresponding single action & observetions."""
+
+    def __init__(
+        self,
+        label: str,
+        qpos_ids: int,
+        qvel_ids: int,
+        act_ids: int,
+        body_fn: typing.Callable = None,
+        bodies: list[int] = [],
```

Comment:
It is not a good idea to use mutable variables as the default value in functions. Can you change to a tuple? `tuple[int, ...] = ()`